dev/com.ibm.ws.microprofile.faulttolerance.metrics/bnd.bnd

```diff
@@ -37,10 +37,17 @@ src: src,resources
 -buildpath: \
         com.ibm.ws.logging;version=latest, \
         com.ibm.ws.logging.core;version=latest, \
+        com.ibm.ws.kernel.service;version=latest,\
         org.eclipse.osgi;version=latest, \
         com.ibm.websphere.org.osgi.core;version=latest, \
         com.ibm.websphere.org.osgi.service.component;version=latest, \
         com.ibm.wsspi.org.osgi.service.component.annotations;version=latest, \
+        com.ibm.ws.microprofile.faulttolerance;version=latest, \
+        com.ibm.ws.microprofile.faulttolerance.spi;version=latest, \
+        com.ibm.websphere.org.eclipse.microprofile.metrics.1.1.1;version=latest, \
+        com.ibm.websphere.org.eclipse.microprofile.config.1.2.1;version=latest,\
```

Comment:
is 1.2.1 the minimum version of config supported?

Reply:
should the package imports specify a range so that config 1.3 and beyond will work?